server/service/client_appconfig.go

```diff
@@ -37,7 +37,7 @@ func (c *Client) ApplyAppConfig(payload interface{}) error {
 }
 
 // GetAppConfig fetches the application config from the server API
-func (c *Client) GetAppConfig() (*fleet.AppConfig, error) {
+func (c *Client) GetAppConfig() (*fleet.EnrichedAppConfig, error) {
```

Comment:
It was only decoding the `fleet.AppConfig` part of the response, I changed it to have all the extra information that the API returns.